packages/manager/src/store/image/image.helpers.ts

```diff
@@ -19,17 +19,7 @@ export const filterImagesByType = (
   }, {});
 };
 
-export const isLinodeKubeImage = (image: Image) =>
-  image.label.match(/kube/i) && image.created_by === 'linode';
-
-export const filterOutKubeImages = (
-  images: Record<string, Image>
-): Record<string, Image> => {
-  return Object.keys(images).reduce((acc, eachKey) => {
-    if (!isLinodeKubeImage(images[eachKey])) {
-      acc[eachKey] = images[eachKey];
-    }
-
-    return acc;
-  }, {});
+// Image IDs are in the form linode/debian11
+export const isLinodeKubeImageId = (id: string) => {
+  return id.startsWith('linode/') && Boolean(id.match(/kube/i));
```

Comment:
Do we think this is sufficient to detect if an Image is a LKE image?

Reply:
I think this is fine for now. We can use the `test` function instead of `match` so that we don't have to cast the result as a boolean

```suggestion
  return id.startsWith('linode/') && /kube/i.test(id);
```